conans/test/integration/generators/cpp_info_set_generator_properties_test.py

```diff
@@ -78,7 +78,7 @@ def package(self):
             def package_info(self):
                 self.cpp_info.set_property("cmake_file_name", "AnotherFileName")
                 self.cpp_info.components["mycomponent"].set_property("cmake_target_name", "mycomponent-name-but-different")
-                self.cpp_info.components["mycomponent"].set_property("cmake_build_modules", [os.path.join("lib", "non-existing.cmake")])
+                self.cpp_info.components["mycomponent"].set_property("cmake_build_modules", [os.path.join("lib", "non-existing.cmake")], path=True)
```

Comment:
I don't like this much. The user will forget to set the `path=True`. We should try to do the path conversions in the consumer (CMakeDeps generator)